pkgs/applications/misc/xygrib/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, cmake, bzip2, qt5, libnova, proj, libpng, openjpeg, makeWrapper } :
```

Comment:
see https://nixos.org/nixpkgs/manual/#ssec-qt-applications: you should not have qt5 as an argument here, and use `libsForQt5.callPackage` instead of `callPackage`